<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewItem</class>
 <widget class="QDialog" name="NewItem">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Item</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources.qrc">
    <normaloff>:/resources/images/icon2.png</normaloff>:/resources/images/icon2.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>265</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="itemNameLabel">
      <property name="text">
       <string>Item Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="itemName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="timeleftLabel">
      <property name="text">
       <string>Time Left</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="descriptionShortLabel">
      <property name="text">
       <string>Short Description</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="descriptionShort"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="description">
      <property name="font">
       <font>
        <family>Tahoma</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="daysLabel">
        <property name="text">
         <string>Days</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="days">
        <property name="maximum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hoursLabel">
        <property name="text">
         <string>Hours</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="hours">
        <property name="maximum">
         <number>23</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minutesLabel">
        <property name="text">
         <string>Minutes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="minutes">
        <property name="maximum">
         <number>59</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>77</width>
     <height>70</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="radioButtonBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>91</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="title">
    <string>Rarity</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>76</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rarityNormal">
       <property name="text">
        <string>Normal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rarityMagic">
       <property name="text">
        <string>Magic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rarityRare">
       <property name="text">
        <string>Rare</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="raritySet">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rarityLegendary">
       <property name="text">
        <string>Legendary</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
